api/envoy/extensions/filters/http/cache/v3alpha/cache.proto

```diff
@@ -19,7 +19,7 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: HTTP Cache Filter]
 
-// [#extension: envoy.filters.http.cache.simple_http_cache]
+// [#extension: envoy.filters.http.cache]
```

Comment:
@htuch not sure if this is correct 

i never really understood why but the cache proto is defined in `/source` and in the api its defined as `simple_http_cache`

if we dont build the v2 protos `envoy.filters.http.cache` is no longer available and building fails